### PR TITLE
Remove examples from readme

### PR DESCRIPTION
**Pull request by JackDevine**
Now that the examples are in the documentation, they are not really needed on the readme. The one example left is just a little fun and it also helps to steer people wanting to do plotting in the right direction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,196 +2,39 @@
...
-using GraphRecipes
-using Plots
-default(size=(1000, 1000))
 
-plot(TreePlot(d), method=:tree, fontsize=10, nodeshape=:ellipse)
+### Primary author: Thomas Breloff (@tbreloff)
 
-```
-![](assets/julia_dict_tree.png)
+This repo maintains a collection of recipes for graph analysis, and is a reduced and refactored version of the previous PlotRecipes. It uses the powerful machinery of [Plots](https://github.com/JuliPlots/Plots.jl) and [RecipesBase](https://github.com/JuliaPlots/RecipesBase.jl) to turn simple transformations into flexible visualizations. Recently, the JuliaPlots organization has taken over the maintenance of GraphRecipes, so contacting them (via opening an issue on this repo) will be a faster way to resolve issues than contacting Tom Brellof.
```
Comment (mkborregaard) [inline README.md]: Just drop the last sentence and the headline above and write `Original author: Thomas Breloff` in normal letters at the very end imho

Comment (JackDevine) [reply README.md]: Done

Comment (mkborregaard) [reply README.md]: Nice. Do you have any position on having your own name on there as primary maintainer? It's not something we've done in the org but maybe we should talk about it

Comment (JackDevine) [reply README.md]: Hi @mkborregaard it feels pretty special that you are suggesting that I put my name on the readme!

Having said that, I have thought about this for a bit and I have decided that I am not in favour of having my name on the readme.

I already get the recognition that I need from the JuliaPlots members and from the data that git stores. Furthermore, if people have a problem, then creating issues/asking for help through the usual channels would be more efficient than asking me directly.

**Pull request merged**